### ISSUE TITLE
A11y_VisualStudioCodeClient_Debugging_StatusMessage:Screen Reader is not announcing about the current status of action in the code editor window

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141873]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #StatusMessages; #WCAG2.1; #AINotInScope; #FTP; #STP;

### Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

### Repro Steps:
1.Open Visual studio code editor.
2.Using "Ctrl+Shift+D" key navigate to the debug side view bar option.
3.Open the launch.json file under the "Configure or Fix: launch.json"button and select "Node.js" from the drop down list
4. Add configurations to "launch.json" utilizing IntelliSense with CTRL+SPACE
5. Parse launch.json with keyboard and observe if ATs will provide stock debugging information under "configurations" properties.
6.Start debugging by pressing F5.
7. Set a breakpoint and pause exceution at the breakpoint by issueing an request/command to your Node process that triggers the line you got the breakpoint
8.Toggle/add/remove breakpoints
9.Verify whether screen reader is announcing the current status of action in the code editor window or not.

### Actual Result:
No Confirmation of action is heard when made changes in the code editor window.

### Expected Result:
There should be a confirmation of action informing user of the current state.

### User Impact:
Screen reader users will be confused if screen reader does not announce the current action/edition in the code.
![A11y_VisualStudioCodeClient_Debugging_StatusMessage](https://user-images.githubusercontent.com/98530069/151790556-664fdd7a-2c4a-431e-b22c-a9a7bf9cc5da.gif)
